### DISCUSSION TITLE
paperless-ngx not importing documents

### DISCUSSION THREAD

**Discussion by joaofgf** [tteck/Proxmox#2490] [General]
After updating or installing the current available version, the paperless-ngx app does not import documents and returns the error: `MissingDependencyError: gs`

**Comment by tteck** [comment]
What Linux distribution are you using? 

https://github.com/paperless-ngx/paperless-ngx/discussions/4925

**Reply by joaofgf** [reply]
the default.

**Reply by tteck** [reply]
Please report the Linux distribution you are using (not default).

**Reply by tteck** [reply]
https://github.com/tteck/Proxmox/issues/2500

**Reply by joaofgf** [reply]
`Debian GNU/Linux 11 (bullseye)`

**Reply by tteck** [reply]
Debian 12 required.

**Comment by joaofgf** [comment]
[https://github.com/paperless-ngx/paperless-ngx/discussions/4925](url)

doesnt work for updating.

**Reply by tteck** [reply]
What version of Ghostscript does your system have?
Use the command `gs`
```
GPL Ghostscript 10.0.0 (2022-09-21)
```

**Reply by joaofgf** [reply]
`GPL Ghostscript 9.53.3 (2020-10-01)`

**Reply by tteck** [reply]
https://github.com/tteck/Proxmox/issues/2500

**Comment by joaofgf** [comment]
![image](https://github.com/tteck/Proxmox/assets/1038868/b35f17a0-52cf-421a-bd05-398e3a0ce5c0)

**Reply by tteck** [reply]
https://github.com/tteck/Proxmox/issues/2500

**Reply by joaofgf** [reply]
thank you i will try. be aware that when installing a new lxc with your script the exact same thing happens.

**Reply by tteck** [reply]
With Debian 12?

**Reply by joaofgf** [reply]
with the default options.

**Comment by joaofgf** [comment]
it worked. thank you very much.
